src/bindings/calib/cameracalibration_bindings.cpp

```diff
@@ -81,17 +81,15 @@ void init_cameracalibration(py::module_ &m) {
             translation_y : float
                 Translation in y axis of calibration file
             )")
-        //.def("rectify", &I3DR::Phase::CameraCalibration::rectify)
-        .def("remapPoint", &I3DR::Phase::CameraCalibration::remapPoint, R"(
-            Remap point
+        .def("rectify", &I3DR::Phase::CameraCalibration::rectify, R"(
```

Comment:
In the C++ Phase the rect_image input is filled with the rectifed image. This is passed as reference so I wasn't sure if this would work with the numpy conversions. Can you manually check that the rect_image is updated when input to this function?